src/exodus/rpctx.cpp

```diff
@@ -1635,6 +1638,13 @@ UniValue exodus_sendmint(const UniValue& params, bool fHelp)
 
     // check error and return the txid (or raw hex depending on autocommit)
     if (result != 0) {
+        for (auto const &id : ids) {
+            try {
+                wallet->EraseSigmaMint(id);
+            } catch (std::runtime_error const &e) {
+                LogPrintf("%s : Fail to erase sigma mints, %s", __func__, e.what());
```

Comment:
Need to add line feed at the end of string.

Reply:
Nice catch